glean-core/src/upload/directory.rs

```diff
@@ -71,7 +76,13 @@ impl PingDirectoryManager {
     ///
     /// Won't panic if unable to delete the file.
     pub fn delete_file(&self, uuid: &str) {
-        let path = self.get_dir().join(uuid);
+        let path = match self.get_file_path(uuid) {
+            Some(path) => path,
+            _ => {
```

Comment:
Personal preference: I would always list out the `None` here instead of a catch-all underscore.